packages/components/src/DocumentViewer/components/PanViewer.tsx

```diff
@@ -51,11 +68,29 @@ const PanViewer: React.FC<IProps> = ({ image, zoom, rotation }) => {
         rotation={rotation}
         key={dx}
       >
-        <img
-          src={image}
-          alt="Supporting Document"
-          style={{ transform: `rotate(${rotation}deg)` }}
-        />
+        {!isPdf ? (
+          <img
+            src={image}
+            alt="Supporting Document"
+            style={{ transform: `rotate(${rotation}deg)` }}
+          />
+        ) : (
+          <Button
+            id="preview_close"
+            aria-label="Preview PDF1970
+             in new tab"
+            size="medium"
+            type="positive"
+            onClick={handleOpenPdf}
+          >
+            {intl.formatMessage({
+              id: 'review.panViewer.openPDF',
```

Comment:
Is there any corresponding country config pr where this/these translations are added in client.csv? if there is please update the country config pr url in this pr's description.